src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -95,6 +95,14 @@ public JsonObject asJson() {
     return representation.copy();
   }
 
+  public JsonObject asJsonWithUserAndProxy() {
+    JsonObject json = asJson();
+    write(json, "item", JsonObject.mapFrom(item));
+    write(json, "user", JsonObject.mapFrom(user));
+    write(json, "proxy", JsonObject.mapFrom(proxy));
+    return json;
+  }
```

Comment:
Why does this need to be converted to JSON in this way?

Reply:
The answer is here https://github.com/folio-org/mod-circulation/pull/683#discussion_r493052737